test/e2e/reconciler/clusterworkspacedeletion/controller_test.go

```diff
@@ -201,12 +201,12 @@ func TestWorkspaceDeletionController(t *testing.T) {
 
 				// get clients for the right shards. We have to access the shards directly to see object (Namespace and ClusterWorkspace) deletion
 				// without being stopped at the (front-proxy) gate because the parent workspace is already gone.
-				rootShardConfig := framework.ShardConfig(t, server.kcpClusterClient, "root", server.RunningServer.DefaultConfig(t))
+				rootShardConfig := framework.ShardConfig(t, server.kcpClusterClient, "root", server.RunningServer.BaseConfig(t))
```

Comment:
can we drop this?

Reply:
oh, this is an old name, i didn't touch it and in fact it uses `BaseConfig()`, not `RootShardConfig()`

Reply:
nvmd. This uses BaseConfig, not rootCfg